### PR TITLE
Fix package.json to run script subcommands sequentially

### PR DESCRIPTION
**Pull request by rensftw**
### What was a problem?
Subcommands in a few `package.json` scripts are being run in two separate processes (first one in a background process and the second one in a standard process)

### How this PR fixes the problem?
Subcommands are now being run sequentially

### CODE REVIEW

File: packages/recomponents/src/components/r-radio/r-radio.spec.js [hunk 1]
```diff
@@ -2,8 +2,6 @@ import {mount, shallowMount} from '@vue/test-utils';
 import {renderToString} from '@vue/server-test-utils';
 import RRadio from './r-radio.vue';
 
-const $t = () => 'custom label';
```
Comment (andrey-rebilly) [inline packages/recomponents/src/components/r-radio/r-radio.spec.js]: What's that?

Comment (rensftw) [reply packages/recomponents/src/components/r-radio/r-radio.spec.js]: This was causing a linting error for an unused const :)

Comment (followyouhome) [reply packages/recomponents/src/components/r-radio/r-radio.spec.js]: old mock for i18n lib

**Pull request merged**